src/apps/bm_devkit/bmdk_common/app_main.cpp

```diff
@@ -137,6 +137,7 @@ SerialHandle_t usbPcap = {
 };
 
 cfg::Configuration *userConfigurationPartition = NULL;
+cfg::Configuration *sysConfigurationPartition = NULL;
```

Comment:
This is the method that `user_code.cpp` gets acces to the cfg partitions, so I added the sys so we can access these in `bcmp_configs_wrapper.cpp`.

Reply:
I feel as if I do not love these being global to app_main.
Could they somehow be instantiated in a source file somewhere with a getter API?